### ISSUE TITLE
Cannot install on Windows/MinGW v0.4.0

### ISSUE THREAD

**Issue by sewbacca** [lunarmodules/luasystem#25]
Installing luasystem v0.4.0 or scm-0 via luarocks causes a compilation error on Windows/MinGW:
```
luarocks install luasystem 0.4.0 --server=https://luarocks.org/dev
Installing https://luarocks.org/luasystem-0.4.0-1.src.rock

luasystem 0.4.0-1 depends on lua >= 5.1 (5.4-1 provided by VM)
gcc -O2 -fPIC -c -o src/core.o -IC:/msys64/ucrt64/include src/core.c -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -IC:\msys64\ucrt64/include -IC:\msys64\ucrt64/include
gcc -O2 -fPIC -c -o src/compat.o -IC:/msys64/ucrt64/include src/compat.c -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -IC:\msys64\ucrt64/include -IC:\msys64\ucrt64/include
gcc -O2 -fPIC -c -o src/time.o -IC:/msys64/ucrt64/include src/time.c -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -IC:\msys64\ucrt64/include -IC:\msys64\ucrt64/include
gcc -O2 -fPIC -c -o src/environment.o -IC:/msys64/ucrt64/include src/environment.c -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -IC:\msys64\ucrt64/include -IC:\msys64\ucrt64/include
gcc -O2 -fPIC -c -o src/random.o -IC:/msys64/ucrt64/include src/random.c -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -IC:\msys64\ucrt64/include -IC:\msys64\ucrt64/include
gcc -O2 -fPIC -c -o src/term.o -IC:/msys64/ucrt64/include src/term.c -DWINVER=0x0600 -D_WIN32_WINNT=0x0600 -IC:\msys64\ucrt64/include -IC:\msys64\ucrt64/include
src/term.c: In function 'lst_readkey':
src/term.c:762:10: error: implicit declaration of function '_kbhit' [-Wimplicit-function-declaration]
  762 |     if (!_kbhit()) {
      |          ^~~~~~
src/term.c:766:18: error: implicit declaration of function '_getwch'; did you mean '_getcwd'? [-Wimplicit-function-declaration]
  766 |     wchar_t wc = _getwch();
      |                  ^~~~~~~
      |                  _getcwd

Error: Build error: Failed compiling object src/term.o
```
Installing 0.3.0 compiles without issue.

**Comment by Tieske** [comment]
I assume that is because of using Msys. After looking around a bit, you might have to include a header called `conio.h` on Msys.

**Comment by Tieske** [comment]
Thinking again; for msys2, it might be better to use the Posix code branches instead.

**Comment by sewbacca** [comment]
Including conio.h compiles. How would I test the Posix code branches?

**Comment by Tieske** [comment]
Fixed by #27

**Comment by Tieske** [comment]
fix released in v0.4.1
